### ISSUE TITLE
file maker recording: retire bugs and features

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2816]
I don't have the feeling anybody is using this ever. So I'd like to retire the bugs & features button in the file maker database. 

@arch-kiosk/test Thoughts?

**Comment by lbestock** [comment]
If none of the field projects used it this summer, then by all means, this makes sense to me.

**Comment by luizaogs** [comment]
Wasn’t the button how Leslie’s team told you about bugs? But I mean, I don’t use it and know most projects don’t so…

**Comment by urapadmin** [comment]
true, but they used it only in their first season and not with much success. The notes were so cryptic or moody that I never really got anything substantial out of it.

**Comment by urapadmin** [comment]
### recording app 16.17
- retired. The cannon button is a thing of the past. Nobody can shoot the developer anymore.

**Comment by lbestock** [comment]
Find mentions of it in the documentation and remove them before closing this ticket.

**Comment by luizaogs** [comment]
Done.
